app/backend/app.py

```diff
@@ -288,6 +288,14 @@ async def setup_clients():
     )
 
 
+@bp.after_app_serving
```

Comment:
This was already an issue that @mattmsft discovered, that we weren't properly closing the clients. It became obvious when I used app in the evaluation module, however, so I'm fixing it here.